Steps/Steps.js

```diff
@@ -94,6 +94,17 @@ const StepsBase = kind({
 		 */
 		futureIcon: PropTypes.string,
 
+		/**
+		 * Defines how to represent the current step.
+		 *
+		 * When `true`, highlight effect will be applied only to the current icon.
+		 * When `false`,highlight effect will be applied to past and current icons.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		highlightCurrentOnly: PropTypes.bool,
```

Comment:
I guess we need to add a change log for adding this as public.

Reply:
It's added.